src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Services/AssessorOverviewOrchestrator.cs

```diff
@@ -27,25 +30,34 @@ public async Task<AssessorApplicationViewModel> GetOverviewViewModel(GetApplicat
                 return null;
             }
 
-            var viewmodel = new AssessorApplicationViewModel(application);
-            viewmodel.Sequences = await _applyApiClient.GetAssessorSequences(application.ApplicationId);
+            var sequences = await _applyApiClient.GetAssessorSequences(application.ApplicationId);
+            if (sequences is null)
+            {
+                return null;
+            }
+
+            var assessorType = AssessorType.FirstAssessor; // SetAssessorType(application, request.UserId);
```

Comment:
Is this always the case?

Reply:
I added a comment and a question back to Anatoli as to why Assessor needs to calculate the assessor type when it should be part of the ViewModel response

Reply:
Here is exactly the point where we have to determine the AssessorType from the UserId and application (Assessor1UserId & Assessor2UserId). Today I am going to work on that SetAssessorType() function.